examples/5.fat_sat_low_field/CMRA_signal_dynamics.jl

```diff
@@ -31,7 +32,7 @@ for i=eachindex(spin_group)
     add_trace!(p0, p1, row=1, col=1)
     add_trace!(p0, p2, row=2, col=1)
 end
-seqd = KomaMRICore.discretize(seq; sim_params)
+seqd = KomaMRICore.discretize(seq; sampling_params)
```

Comment:
```suggestion
seqd = KomaMRICore.discretize(seq; sampling_params=sim_params)
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/commit/72790d5e927932b0a83b571459e14a4602156038